src/app/components/molecules/Input/Input.sc.ts

```diff
@@ -90,6 +92,12 @@ export const TextField = styled.input<TextFieldProps>`
     width: 100%;
     color: ${({ theme }): string => theme.colorTextBody.primary};
 
+    ${({ colorNegativeAmount, hasNegativeAmountColor }): SimpleInterpolation =>
+        hasNegativeAmountColor &&
+        css`
+            color: ${colorNegativeAmount};
```

Comment:
why is this a variable and not coming from the theme?

Reply:
See previous discussion